front/app/containers/Admin/projects/project/events/edit.tsx

```diff
@@ -53,16 +53,13 @@ interface DataProps {
   remoteEventFiles: GetRemoteFilesChildProps;
 }
 
-interface InputProps {
+interface Props extends DataProps {
   params: {
     id: string | null;
     projectId: string | null;
```

Comment:
string or undefined to be precise 🙂 Probably the same for the id (which I would rename eventId)

Reply:
I'll leave it as-is for now, trying to make as few changes to existing components as possible in the course of this upgrade